.nvmrc

```diff
@@ -1 +1 @@
-10.15.3
+lts/erbium
```

Comment:
This is ok, but the backenders prefer to be explicit with the version. Also, it should depend on the platform where it will be deployed (Heroku in this case) You can see this info there: https://devcenter.heroku.com/articles/nodejs-support